cmd/queue/main.go

```diff
@@ -66,6 +69,11 @@ const (
 	// Single concurency queue depth.  The maximum number of requests
 	// to enqueue before returing 503 overload.
 	singleConcurrencyQueueDepth = 10
+
+	// Exponential backoff parameters for creating a connection to the
+	// stats sink.
+	connMinRetryInterval       = 100 * time.Millisecond
+	connExponentialBackoffBase = 1.3
```

Comment:
These seems to be the common defaults.  Perhaps NewExponentialIntervalFunc should provide them.